source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/source/EmissionSourceType.java

```diff
@@ -41,9 +41,13 @@ public enum EmissionSourceType {
    */
   MEDIUM_COMBUSTION_PLANT(Names.MEDIUM_COMBUSTION_PLANT),
   /**
-   * Cold start emission values.
+   * Cold start parking emission values.
    */
-  COLD_START(Names.COLD_START),
+  COLD_START_PARKING(Names.COLD_START_PARKING),
```

Comment:
think it's not just parking, but more a parking garage? The other case is usually also parking, but then outside (or at least no building effects)